### DISCUSSION TITLE
MYSQL建議配置

### DISCUSSION THREAD

**Discussion by HsuChen88** [1101-GS4719-C/Python-Community-GS4719-C#602] [Q&A] [answered]
如圖，小弟我剛下載MYSQL，但由於第一次下載這類型檔案，不太知道這個怎麼選..
不知道有沒有大神有經驗能提供建議的選項?
謝謝
![image](https://user-images.githubusercontent.com/99878799/168637988-ad37523b-6979-493c-90a0-79c66275d265.png)

**Comment by JSHT** [comment, marked as answer]
這邊列表的應該都是他要執行的東西，根據指示按 Execute 就可以了
